src/app/pages/component-viewer/component-viewer.scss

```diff
@@ -1,20 +1,20 @@
-@import 'constants';
+@use '../../../styles/constants';
```

Comment:
Is `@use` not compatible with? Or I guess, does `sass-loader` not support `stylePreprocessorOptions` with `@use`?

https://github.com/angular/material.angular.io/blob/8583f02ea14213d02a354a09cf82cbe0ab1d311a/angular.json#L73-L75

Reply:
I'm guessing it's a `sass-loader` issue, but I think that I prefer the relative path anyway. I found the shorthand confusing at first.

Reply:
I just tested it out and it does look like `sass-loader` and WebStorm are both happy with using `@use` with `stylePreprocessorOptions`. I'd prefer to continue using these shorter paths if they don't cause some kind of build error.